nixos/modules/services/mail/maddy.nix

```diff
@@ -215,6 +214,39 @@ in {
         '';
       };
 
+      tls = {
+        loader = mkOption {
+          type = types.enum [ "file" "off" ];
```

Comment:
What does limit our ability to implement the other loader? Is it only limited by our ability to include the domain registrar tokens from a file?

Reply:
I would like to stick with only implementing `file` loader in this PR since it will be the only one I'm going to use

Reply:
Okay. And if the `security.acme` will ever see the light of day (I try to refrain myself from doing it for now, because I kinda should study for my A-Levels instead of doing that ;D) we would also have support for a wild variety of DNS Challenge providers.